community/plugins/spring-ai-alibaba-starter-plugin-feishu/src/main/java/com/alibaba/cloud/ai/plugin/feishu/FeiShuService.java

```diff
@@ -0,0 +1,101 @@
+package com.alibaba.cloud.ai.plugin.feishu;
+
+import com.google.gson.JsonParser;
```

Comment:
I see that you have used jackson. The global situation should be unified. There is no need for gson here. What do you think?

Reply:
ok